### ISSUE TITLE
skeletor unable to find virtualenv executable

### ISSUE THREAD

**Issue by montaropdf** [chrisbarrett/skeletor.el#40]
Hello,

I am trying to create a `python3` project using skeletor, but it keeps saying that `virtualenv` is not install, while virtualenv can be found by the command-line tool `which` .

```
roland  tanko  ~  $  which virtualenv
virtualenv-3                 virtualenv-clone-3           virtualenvwrapper-3.sh       virtualenvwrapper_lazy_load  
virtualenv-3.6               virtualenv-clone-3.6         virtualenvwrapper_lazy-3.sh  virtualenvwrapper_load       
```

Could it be that skeletor is searching for `virtualenv` explicitely and doesn't accept the suffix -3 and -3.6?

My system is a Linux Fedora 28.

**Comment by chrisbarrett** [comment]
Hey there!

Yep, if you look at the makefile in the project template [here](https://github.com/chrisbarrett/skeletor.el/blob/master/project-skeletons/python-library/Makefile), you'll see it's calling `virtualenv` explicitly.

If you need to use virtualenv-3, you could get around this by copying and modifying the python-library template.

**Comment by montaropdf** [comment]
Do you think it could be relatively easy to tell skeletor to discover all the versions of virtualenv available on the system and ask the user to pick one?

**Comment by chrisbarrett** [comment]
Yeah, that shouldn't be too hard to hack together. I haven't touched this project in a few years, so you're welcome to take a stab at a PR. :)

In the meantime, you should be able to adapt the existing template to get by.

**Comment by montaropdf** [comment]
I have already created a skeleton and some code for those improvement.

I just need to discover how to generate a PR and submit it.

I will try to provide my modifications in the coming weeks.
